src/dpx.imageio/libdpx/WriterInternal.h

```diff
@@ -203,7 +203,11 @@ namespace dpx
 	// this routine expects a type of U16
 	template <typename IB, Packing METHOD>
 	void WritePackedMethodAB_10bit(IB *src, IB *dst, const int len, const bool reverse, BufferAccess &access)
-	{	
+    {
+        OIIO_DASSERT(len >= 1);
+        if (len < 1)
+            return;
+
```

Comment:
Looks like `libdpx` is formatted with tabs and you're space here.  Is clang-format excluded for libs like this?

Reply:
Yes, this is "imported" code that all along I excluded from clang-format so that it remains possible to diff it against the original and incorporate any fixes, should that original library ever be updated. (Though to the best of my knowledge, it's been unmaintained for a long time, so maybe we should give up the pretense?)

But yeah, in vscode it looks like it matches the surrounding text, but I think it looks odd in the github review diff because of space vs tabs differences.